code-of-conduct.md

```diff
@@ -1,38 +1,134 @@
+include:
+
+- Being kind to others
+- Behaving professionally
+- Using welcoming and inclusive language
+- Being respectful of differing viewpoints and experiences
+- Gracefully accepting constructive criticism
+- Focusing on what is best for the community
+- Showing empathy towards other community members
+
+Examples of unacceptable behavior by participants include:
+
+- Continued one-on-one communication after requests to cease
+- Deliberate misgendering, using inappropriate pronouns, or use of “dead” or
+rejected names
+- Harassing or non-consensual photography or recording
+- Harassment of event participants in any form
+- Hate symbols
+- Incitement of violence towards any individual or group, including encouraging
+a person to engage in self-harm
+- Insults, put downs or excessive swearing
+- Intimidation, stalking or following
+- Inappropriate physical contact or unwelcome contact of any kind, especially
+after a request to stop
+- Physical assault and/or battery
+- Policing bathrooms based on gender expression or identity
+- Publication of non-harassing private communication
+- Sexist, racist, homophobic, transphobic, ableist, or other exclusionary jokes
+- Sexual language and imagery in any event venue, including talks, Slack,
+Twitter, or other online media
+- Sustained disruption of talks or other events
+- Threats - violent or otherwise - directed against another person
+- Unwelcome comments regarding a person’s lifestyle choices and practices,
+including those related to food, health, parenting, relationships, drugs, and
+employment
+- Unwelcome sexual attention or advances
+- Other conduct that is inappropriate for a professional audience that includes
+people of many different backgrounds
+
+Participants asked to stop any inappropriate behavior are expected to comply
+immediately.
+
+If a participant engages in behavior that violates this code of conduct, the
+event organizers may take any action they deem appropriate, including warning
+the offender or expulsion from the event (without refund, if applicable) and
+banning from future events.
+
+Thank you for helping make this a welcoming, friendly community for all.
+
+### Weapons Policy
+
+No weapons are allowed at ChiPy venues, including but not limited to explosives
+(including fireworks), guns, and large knives such as those used for hunting or
+display, as well as any other item used for the purpose of causing injury or
+harm to others. Anyone seen in possession of one of these items will be asked
+to leave immediately, and will only be allowed to return without the weapon.
+
+Attendees are further expected to comply with all state and local laws on this
+matter.
+
+## Scope
+
+All ChiPy attendees are subject to the Code of Conduct. This includes ChiPy
+organizers and volunteers, as well as attendees of the talk events,
+collaborative work environments, receptions / after-party events, or other
+sponsored or unofficial events related to ChiPy events.
```

Comment:
Could we clarify what unofficial Chipy event would mean?

Reply:
I don't think so, the point of it is to leave space for someone who does something following a ChiPy event or before. E.g. If I say this in slack `Hey, if you're going to ChiPy, join me at this place first`, we want reserve the right to say folks complaining about conduct at that event could be heard. More specificity removes the blanket application.